polaris-core/src/test/java/org/apache/polaris/service/storage/StorageLocationTest.java

```diff
@@ -19,17 +19,41 @@
 package org.apache.polaris.service.storage;
 
 import org.apache.polaris.core.storage.StorageLocation;
-import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Why not `assertj`? They are generally more user-friendly, IMHO

Reply:
Earlier, I was testing some assertionThrows and JUnit was much easier to work with for those imo. But now that those are gone, I agree for these cases - I'll change them back.